Shared/Samples/Trace utility network/TraceUtilityNetworkView.Model.swift

```diff
@@ -239,30 +239,31 @@ extension TraceUtilityNetworkView {
             }
         }
         
-        /// Updates the textual user hint. 
-        /// 
+        /// Updates the textual user hint.
+        ///
         /// If no message is provided a default hint is used.
         /// - Parameter message: The message to display to the user.
         func updateUserHint(withMessage message: String? = nil) {
-            if let message {
-                hint = message
-            } else {
-                switch tracingActivity {
-                case .none:
-                    hint = ""
-                case .settingPoints(let pointType):
-                    switch pointType {
-                    case .start:
-                        hint = "Tap on the map to add a Start Location."
-                    case .barrier:
-                        hint = "Tap on the map to add a Barrier."
+            DispatchQueue.main.async { [weak self] in
```

Comment:
We could probably mark this method as `@MainActor` instead of using dispatch queue.

Reply:
I tried that out but landed on this because it introduced additional complexity elsewhere as they method became async.

Reply:
Hmm it looks like one spot it is getting called is already marked async and the other spot I think we could just wrap this call to this method in a task. @yo1995 What are your thoughts? I think it would be good to advertise us taking advantage of the new swift technology.

Reply:
Ah, actually that works for me. I had tried `Task` with a priority and that still caused issues. Using `Task` without a priority works fine. I'll go that route.

Reply:
I like the current approach.